SwiftBar/Plugin/PluginManger.swift

```diff
@@ -249,6 +249,13 @@ extension PluginManager {
         content.sound = silent ? nil : .default
         content.threadIdentifier = pluginID
 
+        let urlString = open ?? ""
+        let url = URL(string: urlString)
+
+        if url?.host != nil, url?.scheme != nil {
+            content.userInfo = ["url": urlString]
+        }
+
```

Comment:
How about: 

```swift
if let urlString = href, 
   let url = URL(string: urlString) {
   content.userInfo = ["url":urlString]
}
```

Reply:
Changed.
However, the validation of the host and the scheme is still needed. Otherwise invalid URLs can be passed (see the screenshot)

![url](https://user-images.githubusercontent.com/742624/116174132-c9335300-a748-11eb-9318-0abd4a708c00.png)

Because of that I changed it to:

```swift
if let urlString = href,
   let url = URL(string: urlString), url.host != nil, url.scheme != nil
{
    content.userInfo = ["url": urlString]
}
```